modules/apps/foundation/expando/expando-taglib/src/main/resources/META-INF/resources/custom_attribute/page.jsp

```diff
@@ -294,6 +294,65 @@ ExpandoBridge expandoBridge = ExpandoBridgeFactoryUtil.getExpandoBridge(company.
 								</c:when>
 							</c:choose>
 						</c:when>
+						<c:when test="<%= type == ExpandoColumnConstants.GEOLOCATION %>">
+
+							<%
+								JSONObject geolocationJson = JSONFactoryUtil.createJSONObject(value.toString());
+							%>
+
+							<div id="<portlet:namespace/>CoordinatesContainer">
+								<div class="glyphicon glyphicon-map-marker"
+									id="<%= portletDisplay.getNamespace()+"ExpandoAttribute--" + name + "--Location" %>">
+								</div>
+
+								<liferay-map:map-display
+									geolocation="<%= true %>"
+									latitude='<%= geolocationJson.getDouble("latitude", -33.8688) %>'
+									longitude='<%= geolocationJson.getDouble("longitude", 151.2195) %>'
+									name='<%= "ExpandoAttribute--" + name +"--" %>'
+								/>
+							</div>
+
+							<aui:script use="aui-base, liferay-map-base">
+								var inputName = "<%= portletDisplay.getNamespace()+"ExpandoAttribute--" + name + "--" %>";
+
+								var geolocationField = {
+									init: function() {
+										Liferay.MapBase.get(
+											inputName,
+											function(map) {
+												map.on('positionChange', geolocationField.onPositionChange, geolocationField);
+											}
+										);
+									},
+
+									onPositionChange: function(event) {
+										var instance = geolocationField;
+
+										var location = event.newVal.location;
+
+										var inputNode = A.one('[name="' + inputName + '"]');
+
+										inputNode.val(
+											JSON.stringify(
+												{
+													latitude: location.lat,
+													longitude: location.lng
+												}
+											)
+										);
+
+										var locationNode = A.one('#' + inputName + 'Location');
+
+										locationNode.html(event.newVal.address);
+									}
+								}
```

Comment:
Can you add a semicolon ( ; ) here? It's a good practice to put those to prevent errors when minifying the code.